eng/wpf-debug.targets

```
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <!--
+    Copy PropertyGroup and Import item into the csproj file.
+    Update the path, as per the path to repo.
+
+    <PropertyGroup>
+      <WpfRepoRoot>C:\wpf</WpfRepoRoot>
+    </PropertyGroup>
+    <Import Project="$(WpfRepoRoot)\eng\wpf-debug.targets" />
+    -->
+
+  <PropertyGroup>
+    <WpfConfig>Debug</WpfConfig>
```

```suggestion
    <WpfConfig Condition="'$(WpfConfig)'==''">Debug</WpfConfig>
```
This would allow overriding WpfConfig in case you want to use Release assemblies. One exemple would be for benchmarking a change, you would want Release assemblies.

---

Why use `WpfConfig` at all instead of just using `$(Configuration)` ? Is this customization really needed here? 

I don't want to block or delay this PR by bringing up this question - I just want you to think through this and do what you think is best 👍

---

@vatsan-madhavan  Maybe I want to make my Release application use the Debug version wpf framework

---

@vatsan-madhavan WPF is not built as the part of consuming application. Hence, it is possible, that the consuming application may be in DEBUG mode, but, we built WPF in RELEASE and vice-versa.